bundles/org.eclipse.passage.loc.report.core/src/org/eclipse/passage/loc/report/internal/core/ExportService.java

```diff
@@ -18,16 +18,32 @@
 import org.eclipse.passage.loc.yars.internal.api.ReportException;
 
 /**
- * FIXME doc
- * 
- * FIXME: append the rest of over-customers export commands here
+ * <p>
+ * Central interface of the package: here a client finds proper report action.
+ * </p>
  * 
  * @since 0.1
  */
 @SuppressWarnings("restriction")
 public interface ExportService {
 
+	/**
+	 * Command to export agreed information about all customers interested in the
+	 * given {@code products} to the file pointed by the given {@code path}.
+	 * 
+	 * @param products set of {@code product} identifiers of interest
+	 * @param target   output file with exported data
+	 * 
+	 * @since 0.1
+	 */
 	void exportCustomersForProducts(Set<String> products, Path target) throws ReportException;
 
+	/**
+	 * It is required to install a {@linkplain CustomerStorage} to the service prior
+	 * any export action is asked as it is the base source of information to be
+	 * exported.
+	 * 
+	 * @since 0.1
+	 */
 	void installCustomerStorage(CustomerStorage stogage);
```

Comment:
Oh no, please. Let's return some prepared service if needed

Reply:
Do you mean `install` must not be void?